rss_test/rss_list/tests.py

```diff
@@ -15,20 +15,24 @@
 # incorrect feeds
 feed_wrong_date = { 'title': 'c', 'link': 'e', 'published': 'Wed, 31 Feb 2017 07:25:58 -0600'}
 feed_empty_date = { 'title': 'd', 'link': 'f', 'published': ''}
-feed_naive_date = { 'title': 'e', 'link': 'g', 'published': 'Wed, 31 Feb 2017 07:25:58'}
+feed_naive_date = { 'title': 'e', 'link': 'g', 'published': 'Wed, 28 Feb 2017 07:25:58'}
 
 class DateStringConvertionToDatetimeCase(TestCase):
     def test_expected_date_format_strings(self):
         self.assertEqual(to_date(feed1['published']),
                          datetime.datetime.strptime(feed1['published'], "%a, %d %b %Y %H:%M:%S %z").replace(tzinfo=None))
 
-    def test_unexpected_date_format_strings(self):
-        freezer = freeze_time("2017-03-09")
-        freezer.start()
-        self.assertEqual(to_date(feed_wrong_date['published']), datetime.datetime.now(tz=None))
-        self.assertEqual(to_date(feed_empty_date['published']), datetime.datetime.now(tz=None))
-        self.assertEqual(to_date(feed_naive_date['published']), datetime.datetime.now(tz=None))
-        freezer.stop()
+    def test_wrong_date(self):
+        with freeze_time("2017-03-09"):
```

Comment:
freeze_time to fajna funkcja której wcześniej nie znałem. Zamiast ręcznie robić start i stop lepiej jest użyć pythonowego "context managera", który właśnie do tego służy.